src/shared-types/src/AuditLogEvent.test.ts

```diff
@@ -15,8 +15,8 @@ describe("AuditLogEvent", () => {
       const log = new AuditLogEvent(defaultOptions)
       log.addAttribute("test", value)
       const attrValue: string = log.attributes.test as string
-      expect(attrValue).toHaveLength(1000)
-      expect(attrValue.endsWith("xxx...[truncated]")).toBeTruthy()
+      expect(attrValue).toHaveLength(27)
```

Comment:
Do we need to check the length as well as the exact string? I think probably just line 19 would suffice?

Reply:
Good point, not really need to check the length now. Will remove that.